crates/subspace-service/src/lib.rs

```diff
@@ -786,6 +788,27 @@ where
         }),
     );
 
+    if let Some(registry) = config.prometheus_registry().as_ref() {
+        match NodeMetrics::new(
+            client.clone(),
+            client.import_notification_stream(),
```

Comment:
@rahulksnv this was actually slightly incorrect and fixed in https://github.com/subspace/subspace/pull/2018/commits/dfda03c14c3a52998a8e64bfc94c0fa9ee5d5ea3

Reply:
sg thanks